SabreTools.Models/PlayStation4/AppPkgHeader.cs

```diff
@@ -0,0 +1,238 @@
+using System.Runtime.InteropServices;
+
+namespace SabreTools.Models.PlayStation4
+{
+    /// <see href="https://www.psdevwiki.com/ps4/PKG_files"/>
+    [StructLayout(LayoutKind.Sequential)]
+    public class AppPkgHeader
+    {
+        /// <summary>
+        /// 0x7F434E54

```

Comment:
Please add this as a constant in a new constants file alongside this. See other models for examples.